### PR TITLE
Update vaccines with new research.

### PR DESCRIPTION
**Pull request by beshaya**
Includes Johnson & Johnson + moderna/pfizer update.

Updates both Calculator and Risk Tracker.

### CODE REVIEW

File: public/tracker/vaccine_table.csv [hunk 1]
```diff
@@ -1,5 +1,6 @@
 No Vaccine,1.00,1.00,1.00
 "Unknown vaccine, unknown date",0.40,0.40,0.40
-Pfizer,1.00,0.56,0.20
-Moderna,1.00,0.56,0.20
+Pfizer,1.00,0.56,0.10
+Moderna,1.00,0.56,0.10
 AstraZenica,1.00,0.56,0.40
+Johnson&Johnson,1.00,0.33
```
Comment (blanchardjeremy) [inline public/tracker/vaccine_table.csv:6]: Can you add spaces before and after the ampersand?

File: src/locales/en.json [hunk 2]
```diff
@@ -178,6 +178,7 @@
     "risk_group_empty_warning": "Enter your location first.",
     "saved_scenario_loaded_message": "If needed, adjust the following prefilled answers to match your <strong>{{ scenarioName }}</strong> scenario.",
     "alerts": {
+      "vaccines_update_3": "<0>March 30th, 2021:</0> The calculator and Risk Tracker now include support for Johnson & Johnson's vaccine. Pfizer & Moderna's effectiveness have been updated. Learn more <2>here</2>.",
```
Comment (jfslocum) [inline src/locales/en.json:181]: Should we mention increasing the wait time to two weeks per dose?

Comment (blanchardjeremy) [reply src/locales/en.json:181]: I think so. 👍

File: src/data/__tests__/calculate.test.ts [hunk 3]
```diff
@@ -443,13 +443,15 @@ describe('calculate', () => {
...
+      ${'pfizer'}      | ${2}  | ${0.1}
       ${'moderna'}     | ${0}  | ${1}
       ${'moderna'}     | ${1}  | ${0.56}
-      ${'moderna'}     | ${2}  | ${0.2}
+      ${'moderna'}     | ${2}  | ${0.1}
       ${'astraZeneca'} | ${0}  | ${1}
       ${'astraZeneca'} | ${1}  | ${0.56}
       ${'astraZeneca'} | ${2}  | ${0.4}
+      ${'johnson'}     | ${0}  | ${1}
+      ${'johnson'}     | ${1}  | ${0.33}
```
[the remark this answers is not in the document]
Comment (beshaya) [reply src/data/__tests__/calculate.test.ts:454]: No, they can't. You can use the deploy preview posted by netlify to check behavior.

image

**Comment by blanchardjeremy** [comment]
Looks good to me.

File: public/tracker/vaccine_table.csv (hunk 1 above)
Comment (beshaya) [reply public/tracker/vaccine_table.csv]: Fixed it by just adding a 2nd dose number.

File: src/locales/en.json (hunk 2 above)
Comment (beshaya) [reply src/locales/en.json]: Done.

**Comment by blanchardjeremy** [comment]
What do we need to release in the Risk Tracker to correspond with the 7 to 14 day change?

**Pull request merged**